packages/app-staking/src/ManageStake/index.tsx

```diff
@@ -219,7 +219,7 @@ function ManageStake ({ className, controllerAddress, stashAddress, onClose }: P
               accountId={signingAddress}
               extrinsic={extrinsic}
               icon='sign-in'
-              isDisabled={!isValid || isFinalSession}
+              isDisabled={!isValid || !isFinalSession}
```

Comment:
if it's the final session it should be disabled. I think if using `quick_staking` the session might always be final since sessions per era is 2

Reply:
nvm I see you fixed it in the next commit! :D

Reply:
updated this one... and used isFinalSession.valueOf()
Earlier - isFinalSession always would be some object with true/false and hence the pop - up had summit tx button disabled.. fixed it in https://github.com/cennznet/ui/pull/101/commits/0ae2bfd07d172735bd78f15b4efffe618b3b5a6a